### PR TITLE
Use default build script for reportsDashboards, observabilityDashboards, queryWorkbenchDashboards and remove customized build script

### PR DESCRIPTION
**Pull request by rupal-bq**
Signed-off-by: Rupal Mahajan <maharup@amazon.com>

### Description
- removed customized build script for reportsDashboards
- removed customized build script for observabilityDashboards
- removed customized build script for queryWorkbenchDashboards

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by gaiksaya** [comment]
Hi @rupal-bq What do you think about moving this build.sh in the respective repository? Its will be easier to manage too.

File: scripts/components/reportsDashboards/build.sh [hunk 1]
```diff
@@ -66,39 +66,13 @@ fi
 
 [ -z "$OUTPUT" ] && OUTPUT=artifacts
 [ -z "$PLATFORM" ] && PLATFORM=$(uname -s | awk '{print tolower($0)}')
-[ -z "$ARCHITECTURE" ] && ARCHITECTURE=`uname -m`
```
Comment (joshuali925) [inline scripts/components/reportsDashboards/build.sh:69]: in `usage` you can also change to

```bash
echo -e "-a ARCHITECTURE\t[Optional] Build architecture, ignored."
```

**Review by joshuali925 — APPROVED** [review]

**Comment by rupal-bq** [comment]
> Hi @rupal-bq What do you think about moving this build.sh in the respective repository? Its will be easier to manage too. cc: @peterzhuamazon @dblock

Seems good idea to me if many plugins have special use cases to handle. But I am not sure what is required to do so.

File: scripts/components/reportsDashboards/build.sh (hunk 1 above)
Comment (rupal-bq) [reply scripts/components/reportsDashboards/build.sh:69]: Thanks! added this.

**Comment by gaiksaya** [comment]
Just move the script under `scripts` folder. Example https://github.com/opensearch-project/alerting/tree/main/scripts

**Comment by rupal-bq** [comment]
> Just move the script under `scripts` folder. Example https://github.com/opensearch-project/alerting/tree/main/scripts and delete from here

Adding script to dashboards-reporting https://github.com/opensearch-project/dashboards-reporting/pull/21. deleted from here.

**Review by gaiksaya — APPROVED** [review]

**Comment by gaiksaya** [comment]
@zelinh You okay with this change?

**Review by ps48 — APPROVED** [review]

**Review by zelinh — APPROVED** [review]
LGTM.

**Comment by vamsi-amazon** [comment]
@rupal-bq  @zelinh  Does removing the script will automatically trigger default build script. We want to remove for observability Dashboards as well?

**Comment by zelinh** [comment]
> @rupal-bq @zelinh Does removing the script will automatically trigger default build script. We want to remove for observability Dashboards as well?

@vamsi-amazon Our current workflow actually have a list for place to look for the build script. https://github.com/opensearch-project/opensearch-build/blob/af1be01fb3a1275214b4a964d1737b46de69a9da/src/paths/script_finder.py#L61-L76
If it doesn't find the build script from all of those location above, the last choice is to use the default build script.

**Comment by rupal-bq** [comment]
For dashboards-observability, I don't see a build script at https://github.com/opensearch-project/dashboards-observability. 

I will update this PR to remove https://github.com/opensearch-project/opensearch-build/blob/main/scripts/components/observabilityDashboards/build.sh as well so default can be used

**Pull request merged**